stdlib/LinearAlgebra/test/blas.jl

```diff
@@ -553,4 +553,15 @@ Base.stride(A::WrappedArray, i::Int) = stride(A.A, i)
     end
 end
 
+@testset "get_set_num_threads" begin
+    default = BLAS.get_num_threads()
+    @test default isa Integer
+    @test default > 0
+    new=rand(1:Sys.CPU_THREADS)
```

Comment:
`Sys.CPU_THREADS == 4` but with 2 physical cores I can't set more than 2 threads.

Reply:
So I guess, `new = 1` is the only safe thing here?

Reply:
Perhaps `div(Sys.CPU_THREADS, 2)` too? But `1` would test it.